src/hotspot/cpu/riscv/macroAssembler_riscv.cpp

```diff
@@ -793,7 +793,9 @@ void MacroAssembler::call_VM_leaf_base(address entry_point,
                                        Label *retaddr) {
   int32_t offset = 0;
   push_reg(RegSet::of(t1, xmethod), sp);   // push << t1 & xmethod >> to sp
-  mv(t1, entry_point, offset);
+
+  movptr(t1, entry_point, offset, t2);
```

Comment:
Personally, I prefer `movptr2(t1, entry_point, offset, t0);` which will be consistent with the one in the preceding assembler routine `MacroAssembler::emit_static_call_stub()`.